tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/pages/check-answer-page.cy.js

```diff
@@ -4,14 +4,14 @@ describe("Check answers page", () => {
   const url = "/self-assessment";
 
   beforeEach(() => {
-    selectedQuestionsWithAnswers = [];
     cy.loginWithEnv(url);
 
-    navigateToCheckAnswersPage().then(() =>
-      cy.wrap(selectedQuestionsWithAnswers)
-    );
+    cy.clickFirstSection();
+    cy.clickContinueButton();
 
-    cy.log(cy.wrap(selectedQuestionsWithAnswers));
+    cy.navigateToCheckAnswersPage(selectedQuestionsWithAnswers);
+
+    cy.log(selectedQuestionsWithAnswers);
```

Comment:
Do we need the log?

Reply:
I think so, helps a lot when running Cypress locally for debugging

Reply:
Np